jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.py

```diff
@@ -0,0 +1,419 @@
+from astropy.coordinates.builtin_frames import __all__ as all_astropy_frames
+from astropy.coordinates import SkyCoord
+from astropy import units as u
+
+from pyvo.utils import vocabularies
+from pyvo import registry
+from pyvo.dal.exceptions import DALFormatError
```

Comment:
is pyvo considered a dependency (should be added to pyproject.toml) or optional (try/except on the import and then set a disabled message on the plugin instructing to install pyvo in order to use the plugin)? 

Either way, should merging this be blocked waiting for the 1.6 release of pyvo or is it partially functional without?

Reply:
Currently, pyvo is a required dependency, but I'd guess it would depend on y'all and how you want to integrate it whether you want to mark it as an optional or not,

As for the release, @bsipocz, do you have a sense as to when we might expect a pyvo v1.6 release?

Reply:
Not before the IVOA interop. 
However, I plan to do a 1.5.3 bugfix quite soon, would that be sufficient here? Which PR is a blocker for this?

Reply:
The PR in question is my small one that added quantity support for the radius arg: https://github.com/astropy/pyvo/pull/594. I assume that's a 1.6 and not a 1.5.3 thing?

I don't think it's that important to rush; @kecnry do we just want to pin to `main` for the time being?

Reply:
Oh, I can totally interpret that as a bugfix, we should have supported a quantity radius for a long time.

Reply:
🤷 Works for me! Thanks so much Brigitta!

Reply:
Optional, please.

Reply:
ok, I didn't realize that.  In that case, its probably fine to have the pinned version of pyvo maybe with a comment saying that we can remove the pin once our required version of astroquery requires the same version of pyvo (and we can create a tech debt ticket to revisit that down the road)?  Does that sound reasonable to everyone?

Reply:
I'm happy if y'all are happy! I would put the pin in now, but since Brigitta suggested a 1.5.3 bugfix is coming soon, I'll hold off on putting the pin in until the bugfix is out, or else the pip resolver will complain that 1.5.3 doesn't exist